demisto_sdk/commands/common/tests/base_validator_test.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+from demisto_sdk.commands.common.hook_validations.base_validator import \
+    BaseValidator
+from demisto_sdk.commands.common.hook_validations.structure import \
+    StructureValidator
+from demisto_sdk.tests.constants_test import (
+    INVALID_INTEGRATION_WITH_NO_TEST_PLAYBOOK, SCRIPT_WITH_PLAYBOOK,
+    VALID_INTEGRATION_TEST_PATH, VALID_TEST_PLAYBOOK_PATH)
+
+HAS_TESTS_KEY_UNPUTS = [
+    (VALID_INTEGRATION_TEST_PATH, 'integration', True),
+    (INVALID_INTEGRATION_WITH_NO_TEST_PLAYBOOK, 'integration', False)
+]
+
+
+@pytest.mark.parametrize('file_path, schema, expected', HAS_TESTS_KEY_UNPUTS)
+def test_yml_has_test_key(file_path, schema, expected):
```

Comment:
Please document all of your unit tests as described [here](https://panw-global.slack.com/archives/G011E63JXPB/p1587370983267700).